packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1668,6 +1668,28 @@ describe("calcite-combobox", () => {
       expect(chips.length).toBe(1);
     });
 
+    it("should append unknown tag to top of list", async () => {
```

Comment:
I think we can remove this test since it's already covered in the updated tests above.